pages/mesosphere/dcos/services/cassandra/2.7.0-3.11.4/release-notes/index.md

```diff
@@ -7,6 +7,20 @@ menuWeight: 10
 model: /mesosphere/dcos/services/cassandra/data.yml
 render: mustache
 ---
+# Release Notes for Cassandra Service version 2.7.0-3.11.4
+
+## Updates
+
+- Upgraded SDK library to version `0.57.0`. For more information see [SDK Release Notes](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0)
+- Added support for [Secure JMX](/mesosphere/dcos/services/{{ model.serviceName }}/2.7.0-3.11.4/advanced/#secure-jmx-enterprise)
+
+## Upgrading your cluster from 2.5.0-3.11.3 to 2.7.0-3.11.4
+
+- If upgrading from version `2.5.0-3.11.3`, you'll need to run the following command to successfully upgrade your Cassandra package: 
```

Comment:
Let's add a note about how it's possible to skip an upgrade step
```suggestion
- It's possible to upgrade directly to `2.7.0-3.11.4` from version `2.5.0-3.11.3`. However, you'll need to run the following command to successfully upgrade your Cassandra package: 
```

Reply:
Done

Reply:
thanks @SarahStegall